endpoints/events/vtrack.go

```diff
@@ -269,25 +270,34 @@ func getAccountId(httpRequest *http.Request) string {
 	return httpRequest.URL.Query().Get(AccountParameter)
 }
 
-// modifyVastXml modifies BidCacheRequest element Vast XML data
-func modifyVastXml(externalUrl string, data json.RawMessage, bidid string, bidder string, accountId string, timestamp int64) json.RawMessage {
-	c := string(data)
+// modifyVastXmlString rewrites and returns the string vastXML or an empty string if it was left unmodified
+func ModifyVastXmlString(externalUrl string, c string, bidid string, bidder string, accountId string, timestamp int64) string {
```

Comment:
Nitpick: There's no need to repeat type declarations of arguments. This can be simplified to:
```
func ModifyVastXmlString(externalUrl, c, bidid, bidder, accountId string, timestamp int64) string {
```

Reply:
Nice, didn't know that